xrootd/xrdproto/prepare/prepare.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2018 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package prepare contains the types related to the prepare request.
+// See xrootd protocol specification (http://xrootd.org/doc/dev45/XRdv310.pdf, p. 106) for more details.
```

Comment:
It's page 69 actually. (I actually used that link this time!) :)

Reply:
copy-pasta... :)

(apologies if my answers+PR-updates are a bit asynchronous and "out-of-band" with your helpful comments: I am in a train with really intermittent mobile data, and no wifi...)

Reply:
done